x-pack/plugins/observability/public/pages/alerts/render_cell_value.tsx

```diff
@@ -62,48 +61,25 @@ export const getMappedNonEcsValue = ({
  */
 
 export const getRenderCellValue = ({
-  rangeTo,
-  rangeFrom,
   setFlyoutAlert,
 }: {
-  rangeTo: string;
-  rangeFrom: string;
   setFlyoutAlert: (data: TopAlert) => void;
 }) => {
-  return ({ columnId, data, setCellProps }: CellValueElementProps) => {
+  return ({ columnId, data }: CellValueElementProps) => {
     const { observabilityRuleTypeRegistry } = usePluginContext();
     const value = getMappedNonEcsValue({
       data,
       fieldName: columnId,
     })?.reduce((x) => x[0]);
 
-    const theme = useTheme();
-
     switch (columnId) {
       case ALERT_STATUS:
-        switch (value) {
-          case ALERT_STATUS_ACTIVE:
-            return (
-              <EuiHealth color="primary" textSize="xs">
-                {i18n.translate('xpack.observability.alertsTGrid.statusActiveDescription', {
-                  defaultMessage: 'Active',
-                })}
-              </EuiHealth>
-            );
-          case ALERT_STATUS_RECOVERED:
-            return (
-              <EuiHealth color={theme.eui.euiColorLightShade} textSize="xs">
-                <EuiText color="subdued" size="relative">
-                  {i18n.translate('xpack.observability.alertsTGrid.statusRecoveredDescription', {
-                    defaultMessage: 'Recovered',
-                  })}
-                </EuiText>
-              </EuiHealth>
-            );
-          default:
-            // NOTE: This fallback shouldn't be needed. Status should be either "active" or "recovered".
-            return null;
+        if (value !== ALERT_STATUS_ACTIVE && value !== ALERT_STATUS_RECOVERED) {
```

Comment:
Your reasoning makes sense to me but I'm uncertain if "no status" is a valid status. The comment that was in the code stated "This fallback shouldn't be needed. Status should be either "active" or "recovered"." and the type we have in AlertStatus doesn't include null/undefined. 

@afgomez 
How can we confirm if "no status" is valid? If we can confirm that I will gladly make that change since it will be better in the code :)

Reply:
We can check [the mapping](https://github.com/afgomez/kibana/blob/820b1cafa098220bbc0a7a7cb6779c720422f1a2/x-pack/plugins/rule_registry/common/assets/field_maps/technical_rule_field_map.ts#L29), and it is indeed `required`. I wonder then why the types don't reflect that :/

Reply:
I think it's because the shared T-grid code doesn't make any special type checks for the row data being Alerts data.
This is the type that is used https://github.com/afgomez/kibana/blob/820b1cafa098220bbc0a7a7cb6779c720422f1a2/x-pack/plugins/timelines/common/search_strategy/timeline/events/all/index.ts#L27

Reply:
@afgomez I'll be looking at this today but maybe we can fix this in a follow up PR?

Reply:
Yeah it's not that important.